components/properties/src/lib.rs

```diff
@@ -92,3 +93,5 @@ pub use props::{
     BidiClass, CanonicalCombiningClass, EastAsianWidth, EnumeratedProperty, GeneralCategory,
     GeneralCategoryGroup, GraphemeClusterBreak, LineBreak, Script, SentenceBreak, WordBreak,
 };
+
+pub use bidi::BidiClassAdapter;
```

Comment:
Nit: Don't hoist this to the top. Leave it exported under the bidi module.

Reply:
I did not get this, could you illustrate more?

Reply:
Should clients write

```rust
use icu_properties::BidiClassAdapter;
```

or

```rust
use icu_properties::bidi::BidiClassAdapter;
```

I think we should lean toward putting this into a child namespace rather than the top level, but I can be convinced otherwise.

CC @Manishearth @echeran

Reply:
I think child module makes sense